client/my-sites/site-settings/form-general.jsx

```diff
@@ -649,23 +509,19 @@ class SiteSettingsFormGeneral extends Component {
 		];
 
 		return (
-			<FormFieldset>
-				<FormLabel>
-					{ translate( 'Week Starts On' ) }
-				</FormLabel>
-				<FormSelect
-					disabled={ isRequestingSettings }
-					name="start_of_week"
-					onChange={ handleSelect }
-					value={ start_of_week || 0 }
-				>
-					{ daysOfWeek.map( ( day, index ) =>
-						<option key={ index } value={ index } >
-							{ day }
-						</option>
-					) }
-				</FormSelect>
-			</FormFieldset>
+			<Card
+				className="site-settings__date-time-format"
+				href={ `/settings/date-time-format/${ site.slug }` }
+			>
+				<h2 className="site-settings__date-time-format-title">
+					{ translate( 'Date and Time Format' ) }
+				</h2>
+				<div className="site-settings__date-time-format-info">
+					{ date_format ? today.format( phpToMomentDatetimeFormat( date_format ) ) : '' } &bull;&nbsp;
+					{ time_format ? today.format( phpToMomentDatetimeFormat( time_format ) ) : '' } &bull;&nbsp;
+					{ translate( 'Week starts on' ) } { start_of_week ? daysOfWeek[ start_of_week ] : daysOfWeek[ 0 ] }
```

Comment:
Hi! I've found a possible matching string that has already been translated **88** times:
`translate( 'Week Starts On' )` *ES Score: 13.92*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).